From 88e4dbee0468d4995df068233c2c559bc2373960 Mon Sep 17 00:00:00 2001
From: Anne-Sophie LEFEVRE <anne-sophie.lefevre@intel.com>
Date: Thu, 21 Nov 2013 10:28:46 +0100
Subject: [PORT FROM MAIN] link libvpp_setting.so

BZ: 151191

link libvpp_setting.so

Change-Id: Ieb1d839f87784167084c3ff4d574818d714fe5cc
Orig-Change-Id: Ia776f711c04a65ec4978bc23cce6b88005bfc18b
Category: feature differentiation
Domain: Video.Media-VPP
Origin: internal
Upstream-Candidate: no
Signed-off-by: Yu, Linda <linda.yu@intel.com>
---
 media/libmediaplayerservice/Android.mk          |    3 ++-
 media/libmediaplayerservice/nuplayer/Android.mk |    3 ++-
 media/libstagefright/Android.mk                 |    3 ++-
 3 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/media/libmediaplayerservice/Android.mk b/media/libmediaplayerservice/Android.mk
index c3d9d59..535b515 100644
--- a/media/libmediaplayerservice/Android.mk
+++ b/media/libmediaplayerservice/Android.mk
@@ -48,7 +48,8 @@ ifeq ($(TARGET_HAS_VPP),true)
 LOCAL_SHARED_LIBRARIES += libva \
                           libva-android \
                           libva-tpi \
-                          libui
+                          libui \
+                          libvpp_setting
 LOCAL_STATIC_LIBRARIES += libvpp
 endif
 
diff --git a/media/libmediaplayerservice/nuplayer/Android.mk b/media/libmediaplayerservice/nuplayer/Android.mk
index 464f401..9ca5665 100644
--- a/media/libmediaplayerservice/nuplayer/Android.mk
+++ b/media/libmediaplayerservice/nuplayer/Android.mk
@@ -27,7 +27,8 @@ LOCAL_C_INCLUDES += \
         $(TARGET_OUT_HEADERS)/libva	\
 LOCAL_SHARED_LIBRARIES += libva \
                           libva-android \
-                          libva-tpi
+                          libva-tpi \
+                          libvpp_setting
 endif
 
 #ifeq ($(TARGET_HAS_MULTIPLE_DISPLAY),true)
diff --git a/media/libstagefright/Android.mk b/media/libstagefright/Android.mk
index 8380e15..cbcd0ab 100644
--- a/media/libstagefright/Android.mk
+++ b/media/libstagefright/Android.mk
@@ -115,7 +115,8 @@ LOCAL_C_INCLUDES += \
 LOCAL_STATIC_LIBRARIES += libvpp
 LOCAL_SHARED_LIBRARIES += libva \
                           libva-android \
-                          libva-tpi
+                          libva-tpi \
+                          libvpp_setting
 endif
 
 ifeq ($(USE_INTEL_MDP),true)
-- 
1.7.4.1

